File: forge/src/main/java/net/impleri/playerskills/client/forge/PlayerSkillsClientForge.java
package net.impleri.playerskills.client.forge;

import net.impleri.playerskills.client.PlayerSkillsClient;
import net.minecraftforge.api.distmarker.Dist;
import net.minecraftforge.eventbus.api.SubscribeEvent;
import net.minecraftforge.fml.common.Mod;
import net.minecraftforge.fml.event.lifecycle.FMLClientSetupEvent;

@Mod.EventBusSubscriber(value = Dist.CLIENT)
public class PlayerSkillsClientForge {
    @SubscribeEvent
    public static void onClientInit(FMLClientSetupEvent event) {
        PlayerSkillsClient.init();
    }
}

File: forge/src/main/kotlin/net/impleri/playerskills/client/forge/PlayerSkillsClientForge.kt
package net.impleri.playerskills.client.forge

import net.impleri.playerskills.client.PlayerSkillsClient
import net.minecraftforge.api.distmarker.Dist
import net.minecraftforge.eventbus.api.SubscribeEvent
import net.minecraftforge.fml.common.Mod.EventBusSubscriber
import net.minecraftforge.fml.event.lifecycle.FMLClientSetupEvent

@EventBusSubscriber(value = [Dist.CLIENT])
object PlayerSkillsClientForge {
  @SubscribeEvent
  fun onClientInit(event: FMLClientSetupEvent?) {
    PlayerSkillsClient.init()
  }
}
